hail/src/test/scala/is/hail/expr/ir/EmitStreamSuite.scala

```diff
@@ -0,0 +1,301 @@
+package is.hail.expr.ir
+
+import is.hail.annotations.{Region, SafeRow, ScalaToRegionValue}
+import is.hail.asm4s._
+import is.hail.asm4s.joinpoint._
+import is.hail.expr.types.physical._
+import is.hail.expr.types.virtual._
+import is.hail.utils._
+import is.hail.variant.Call2
+
+import org.apache.spark.sql.Row
+import org.scalatest.testng.TestNGSuite
+import org.testng.annotations.Test
+
+class EmitStreamSuite extends TestNGSuite {
```

Comment:
I think this needs to extend `HailSuite` (which extends `TestNGSuite`)

Reply:
sure. what does HailSuite do differently?

Reply:
It does some setup like making sure the hail context is initialized. I tried running the `EmitStreamSuite` myself and got an error because the hail context was uninitialized. It must only happen when running the suite in isolation.